.ko.yaml

```diff
@@ -45,3 +45,250 @@ baseImageOverrides:
   k8s.io/test-infra/gcsweb/cmd/gcsweb: gcr.io/k8s-prow/alpine:v20200713-e9b3d9d
   # prow integration test
   k8s.io/test-infra/prow/test/integration/fakeghserver: gcr.io/k8s-prow/alpine:v20200713-e9b3d9d
+
+builds:
+- id: branchprotector
```

Comment:
How did you choose this subset of the Prow images?

Reply:
it's all of the images, actually forgot to include the images from the distroless set

Reply:
done